src/app/pages/checkout-payment/components/payment-concardis-creditcard/payment-concardis-creditcard.component.ts

```diff
@@ -186,33 +191,40 @@ export class PaymentConcardisCreditcardComponent implements OnInit, OnChanges, O
     this.resetErrors();
     if (error) {
       // map error messages
-      this.errorMessage.cardNumber = error.message.properties.find(prop => prop.key === 'cardNumber');
-      if (this.errorMessage.cardNumber && this.errorMessage.cardNumber.code) {
-        this.errorMessage.cardNumber.messageKey = this.getErrorMessage(
-          this.errorMessage.cardNumber.code,
-          'number',
-          this.errorMessage.cardNumber.message
-        );
-      }
-
-      this.errorMessage.cvc = error.message.properties.find(prop => prop.key === 'verification');
-      if (this.errorMessage.cvc && this.errorMessage.cvc.code) {
-        this.errorMessage.cvc.messageKey = this.getErrorMessage(
-          this.errorMessage.cvc.code,
-          'cvc',
-          this.errorMessage.cvc.message
-        );
-      }
+      if (typeof error.message !== 'string' && error.message && error.message.properties) {
```

Comment:
I think when `typeof error.message !== 'string'` is true, then there has to be an object, so the following `error.message` is true at this point